### PR TITLE
Berokka wrapper

### PR DESCRIPTION
**Pull request by Kurt-Hanel**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

This pull request adds a Galaxy wrapper for [Berokka](https://github.com/tseemann/berokka). This is my first wrapper and pull request. Please let me know if there is anything I can fix!

### CODE REVIEW

File: tools/berokka/berokka.xml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+            --fuzz '${fuzz}'
+
+            #unless $anno:
+                --noanno
+            #end unless
+
+    ]]></command>
+    <inputs>
+        <param name="input_file" type="data"  format="fasta" label="Input (FASTA) -- Should be completed long-read assemblies in FASTA format, such as those from CANU or HGAP"/>
```
Comment (bgruening) [inline tools/berokka/berokka.xml]: For the longer part of the label you could add this into a help=""

File: tools/berokka/berokka.xml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+        <conditional name="filter">
+            <param name="filter_select" type="boolean" checked="false" label="Filter -- Allows you to remove contigs which match 50% of sequences in this file."/>
+                <when value="true"> 
+                    <param name="filter_fasta" type="data" format="fasta" label="Filter (FASTA) -- Give a fasta to use as a filter."/>
```
Comment (bgruening) [inline tools/berokka/berokka.xml]: You can remove this conditional and simply add optional="true" to the filter_fasta param.

File: tools/berokka/berokka.xml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                </when>
+                <when value="false"/>
+        </conditional>
+
+        <param name="read_length" type="integer" value="60000" label="Read Length -- Approximate max read length (default '60000')"/>
```
Comment (bgruening) [inline tools/berokka/berokka.xml]: use help="" for the text after `--`

File: tools/berokka/berokka.xml [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+        <param name="fuzz" type="integer" value="5" label="Fuzz -- Accept local alignment within X bp of global (default '5')"/>
```
Comment (bgruening) [inline tools/berokka/berokka.xml]: help :)

Comment (bgruening) [reply tools/berokka/berokka.xml]: Do you know if this param can be restricted with max/min?

File: tools/berokka/berokka.xml [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+            <param name="filter_select" type="boolean" checked="false" label="Filter -- Allows you to remove contigs which match 50% of sequences in this file."/>
+                <when value="true"> 
+                    <param name="filter_fasta" type="data" format="fasta" label="Filter (FASTA) -- Give a fasta to use as a filter."/>
+                </when>
+                <when value="false"/>
+        </conditional>
+
+        <param name="read_length" type="integer" value="60000" label="Read Length -- Approximate max read length (default '60000')"/>
+        <param name="fuzz" type="integer" value="5" label="Fuzz -- Accept local alignment within X bp of global (default '5')"/>	
+        <param name="anno" type="boolean" checked="true" label="Annotation -- Annotate Trimmed FASTA"/>
```
Comment (bgruening) [inline tools/berokka/berokka.xml]: help?

File: tools/berokka/berokka.xml [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+* `Filter <FASTA>` allows you to remove contigs which match 50% of sequences in this file. Berokka comes with the standard Pacbio control sequence. You can provide your own FASTA file using this option.
+
+* `Read Length <LENGTH>` can be used for datasets that won't seem to circularise. It affects the length of the match it attempts to make using BLAST.
+
+* `Fuzz` can be used to accept local alignment within X bp of global (default '5')
+
+* `Annotation` can be set to "No" to ensure that the FASTA descriptions are not altered between the input and output FASTA files.
+
+    ]]></help>
+    <citations>
```
Comment (bgruening) [inline tools/berokka/berokka.xml]: You could try to cite Torstens github repo. There are a few examples in this repository already that do this.

File: tools/berokka/berokka.xml (hunk 4 above)
Comment (Kurt-Hanel) [reply tools/berokka/berokka.xml]: I found a minimum for it, but there is no max.

Comment (Kurt-Hanel) [reply tools/berokka/berokka.xml]: Fuzz doesn't appear to have a min or max either.

**Comment by bgruening** [comment]
Very nice @Kurt-Hanel! Thanks a lot!

**Comment by Kurt-Hanel** [comment]
Glad things are looking good @bgruening, but it still isn't passing tests on planemo. Please see my new pull request.

**Comment by bgruening** [comment]
My mistake. Nothing happened so far. So you can leave this in if you like and update the version with the DB if you are ready.

Sorry for that.

**Comment by Kurt-Hanel** [comment]
No problem!

**Pull request merged**